src/ResourceManager/AzureBackup/Commands.AzureBackup/Commands.AzureBackup.csproj

```diff
@@ -56,9 +56,9 @@
     <Reference Include="Microsoft.Azure.Common.NetFramework">
       <HintPath>..\..\..\packages\Microsoft.Azure.Common.2.1.0\lib\net45\Microsoft.Azure.Common.NetFramework.dll</HintPath>
     </Reference>
-    <Reference Include="Microsoft.Azure.Management.BackupServicesManagement, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
+    <Reference Include="Microsoft.Azure.Management.BackupServicesManagement, Version=1.0.0.0, Culture=neutral, processorArchitecture=MSIL">
       <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\..\..\packages\Microsoft.Azure.Management.BackupServices.1.0.0-preview\lib\net40\Microsoft.Azure.Management.BackupServicesManagement.dll</HintPath>
+      <HintPath>Resources\Microsoft.Azure.Management.BackupServicesManagement.dll</HintPath>
```

Comment:
I've also sent another PR for azure-sdk-for-net. Once that PR is merged, I will publish the nuget and take dependency on the nuget instead of this DLL. In the meanwhile, can you please continue with the review?